documentation/docs/services/SSI-bridge/use-cases.md

```diff
@@ -0,0 +1,108 @@
+---
+image: /img/integration-services/logo/integration_services.png
+description: In the context of e-commerce and the ENSURESEC project, you can use the SSI-Bridge to verify identities for products, customer, sellers, and clients.   
+keywords:
+- ecommerce
+- scanner device
+- customer
+- credentials
+- decentralized identity
+- shopping app
+- products authenticity credentials
+---
+# Use Cases
+
+In the context of the [ENSURESEC](https://www.ensuresec.eu/) e-commerce ecosystem, the proposed use cases will use the
+Ecommerce-SSI Bridge to implement the following workflows.
+
+## Secure Goods Distribution
+
+### Delivery Company Identity and Scanners Verification
+
+**Problem**: Protection of a Delivery. Company X wants to protect its goods from being handled by unauthorized carriers
+and threats or frauds in the distribution chain.
+
+1. An authorized employee of delivery **company X** uses the Ecommerce-SSI Bridge to register
+   an [identity (DID)](https://wiki.iota.org/identity.rs/decentralized_identifiers/overview) for their organization.
+2. An e-commerce operator verifies the delivery company’s organization identity and uses the Bridge to issue a
+   credential for the company to deliver on their behalf. The credential contains the company organization DID and is
+   signed by the e-commerce operator’s private key which had been previously associated with the operator’s identity.
+3. The authorized employee of the verified delivery company registers a DID for each scanner (i.e., android scanners)
+   used by the company couriers.
+4. The authorized employee uses the Ecommerce-SSI Bridge to issue authorization credentials to the scanner devices used
+   to handle deliveries. These credentials are stored locally in the scanner devices.
+5. When a courier hands over the delivery, the scanner device uses the Ecommerce-SSI Bridge to present its credential to
+   the e-commerce operator.
+6. The e-commerce operator uses the Ecommerce-SSI Bridge to verify that the parcel was only handled by an authorized
+   courier, nor was it stolen or diverted in transit. This is possible because of the verification of the device
```

Comment:
The usage of nor is not quite proper, it should rather be something like this:

```suggestion
   courier and was not stolen or diverted in transit. This is possible because of the verification of the device
```

Reply:
```suggestion
   courier, and it was not stolen or diverted in transit. This is possible because of the verification of the device
```